### ISSUE TITLE
Reboots are a problem for alarm

### ISSUE THREAD

**Issue by hendriku** [dhbw-timetable/dhbw-timetable-android#83]
They aren't rescheduled :(

**Comment by d-Rickyy-b** [comment]
This is because you just start the service here instead of scheduling an alarm:

https://github.com/dhbw-timetable/dhbw-timetable-android/blob/2a5246a9f749e915b8f0d7a6bf2f539262b907ed/app/src/main/java/dhbw/timetable/services/TimetableRestarterBroadcastReceiver.java#L10-L14

Instead you should use the AlarmManager and pass it the service as pending intent similar to this:
```
Intent intentsOpen = new Intent(context, BackgroundService.class);
PendingIntent pendingIntent = PendingIntent.getService(context, 0, intentsOpen, 0);
AlarmManager alarmManager = (AlarmManager) context.getSystemService(Context.ALARM_SERVICE);
alarmManager.setRepeating(AlarmManager.RTC_WAKEUP, firstStart, interval, pendingIntent);
```

Or use your [addAlarm method](https://github.com/dhbw-timetable/dhbw-timetable-android/blob/b4d693aaee6d472917392eb8ed30948301a58bc7/app/src/main/java/dhbw/timetable/navfragments/notifications/alarm/AlarmSupervisor.java#L181-L184)

**Comment by hendriku** [comment]
@d-Rickyy-b Thanks for your suggestion, you did guess right it was a BroadcastReciever, but not the `TimetableRestaterBroadcastReceiver.java`. This Receiver is just for restarting the background sync service. It is meant for `.services.ActivityRecognition.RestartSensor`. I did need to enable the `DeviceBootReceiver.java` which listenes for `android.intent.action.BOOT_COMPLETED`.
I'll distribute this fix in the new v1.2.

**Comment by d-Rickyy-b** [comment]
Nice, glad you were able to resolve it :) 👍
